Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-shadertoy/) for Kodi add-ons: look and feel/visualization.shadertoy.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-look-and-feel/-/visualization-shadertoy/horizontal-auto.svg)

gade01 commented:
Contains wrong translations:
https://github.com/xbmc/visualization.shadertoy/pull/86/files#diff-16eedc0d986c088d6abefdfc7b5c73ed7df33f04c3100fedd323672a72d0f51bR36

https://github.com/xbmc/visualization.shadertoy/pull/86/files#diff-16eedc0d986c088d6abefdfc7b5c73ed7df33f04c3100fedd323672a72d0f51bR46

etc.

weblate committed:
Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 97.2% (35 of 36 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (36 of 36 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: 撷星芒 <mofa2020@qq.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-shadertoy/zh_cn/
Translation: Kodi add-ons: look and feel/visualization.shadertoy

gade01 reviewed (approved):
Everything solved now.
